src/index.js

```diff
@@ -8,30 +8,51 @@ import {runScripts} from './transformScriptTags';
 function processOptions(options) {
   // Parse preset names
   const presets = (options.presets || []).map(presetName => {
-    if (typeof presetName === 'string') {
-      var preset = availablePresets[presetName];
-      if (!preset) {
-        throw new Error(`Invalid preset specified in Babel options: "${presetName}"`);
+    let preset = null;
+
+    if (presetName instanceof Array && typeof presetName[0] === 'string') {
+      if (presetName[0] in availablePresets) {
```

Comment:
Nit: Use `hasOwnProperty` instead of `in`; `in` does a prototype chain check and thus will do strange things in environments where `Object.prototype` has been extended.